.pre-commit-config.yaml

```
@@ -18,7 +18,7 @@ repos:
     language: python
     stages: ["commit"]
     types: [text]
-    exclude: '^tests|NOTES|README|autogen|^(CHANGES|VERSION|scripts/ninja-build-stats|\.gitmodules|\.paths|.*\.(decl|json|rst|t2d|txt))$'
+    exclude: '^tests|NOTES|README|autogen|^(CHANGES|VERSION|scripts/ninja-build-stats|\.gitmodules|\.paths|setup.cfg|.*\.(decl|json|rst|t2d|txt|dot))$'
```

Adding `dot` seems not needed. Can you just add a license header to `setup.cfg`?

```suggestion
    exclude: '^tests|NOTES|README|autogen|^(CHANGES|VERSION|scripts/ninja-build-stats|\.gitmodules|\.paths|.*\.(decl|json|rst|t2d|txt))$'
```